supporting_content/columns/amortizationcategory.md

```diff
@@ -0,0 +1,52 @@
+# Column: Amortization Category
+
+Objective: we need a simple method to calculate savings that doesn't accidentally double-count commitment discount purchase costs.
+Objective: we need a simple method to identify which line items are and are not eligible for
+commitment discounts, and which actually have commitment discounts applied to them.
```

Comment:
This feels separate from amortization. I do think this is something we need, but it needs to account for usage and spend commitment discounts. I think that needs to be a separate column (or 2).